MovieApp/MovieApp/App/Features/MovieBrowser/Components/MovieCellHPv1/MovieCell+Design.swift

```diff
@@ -54,24 +50,30 @@ extension MovieCell: DesignProtocol {
     }
     
     func defineLayoutForViews() {
-        movieImageView.snp.makeConstraints {
-            $0.leading.equalToSuperview()
+        contentView.snp.makeConstraints {
             $0.top.equalToSuperview()
-            $0.height.equalToSuperview()
+            $0.bottom.equalToSuperview().inset(12)
+            $0.leading.equalToSuperview().offset(18)
+            $0.trailing.equalToSuperview().inset(18)
```

Comment:
leading can be inset and you can make it a one liner

Reply:
[Here](https://github.com/lihter/Movie-App/pull/41/commits/3383b62f0e45eb7e1f5ba5b650a01beaa73387a1#diff-4e7b2efbcaaa979e7e77ed095dcb6e1fae870d3f9216aa6bbf6a65457310e589R56)
Also replaced everywhere else where it was possible to apply the same code style fix.